embc-app/Controllers/RegistrationsController.cs

```diff
@@ -91,6 +91,13 @@ public async Task<IActionResult> Create([FromBody] ViewModels.Registration item)
             {
                 return BadRequest(ModelState);
             }
+
+            if (!item.DeclarationAndConsent.HasValue || !item.DeclarationAndConsent.Value)
```

Comment:
If you do this before the line 90, you don't need to return BadRequest, the previous ModelState.IsValid will take care of it

Reply:
So funny I had it like that originally, but I wanted to ensure the Model State was valid or else an exception happens